styleguide.config.js

```diff
@@ -1,4 +1,17 @@
 const path = require('path')
+const CopyWebpackPlugin = require('copy-webpack-plugin')
+const rules = require('./webpack/rules.js')
```

Comment:
Lines 3-14 could live in `webpack/styleguide.rules.js` which in turn could be required on line 47 - this ensures all the webpack config lives in the webpack folder. Let me know if I am missing something?